test/rhsmlib/services/test_register.py

```diff
@@ -271,35 +273,11 @@ def test_register_normally_old_candlepin(self, mock_persist_consumer, mock_write
         self.mock_cp.registerConsumer.return_value = expected_consumer
 
         register_service = register.RegisterService(self.mock_cp)
-        consumer = register_service.register("org", name="name", environments="environment")
-
-        self.mock_cp.registerConsumer.assert_called_once_with(
-            name="name",
-            facts={},
-            owner="org",
-            environments="environment",
-            keys=None,
-            installed_products=[],
-            jwt_token=None,
-            content_tags=[],
-            consumer_type="system",
-            role="",
-            addons=[],
-            service_level="",
-            usage="",
-        )
-        self.mock_installed_products.write_cache.assert_called()
-
-        mock_persist_consumer.assert_called_once_with(expected_consumer)
-        mock_write_cache.assert_called_once()
-        expected_plugin_calls = [
-            mock.call("pre_register_consumer", name="name", facts={}),
-            mock.call("post_register_consumer", consumer=expected_consumer, facts={}),
-        ]
-        self.assertEqual(expected_plugin_calls, self.mock_pm.run.call_args_list)
-        assert "owner" in consumer
-        assert "contentAccessMode" in consumer["owner"]
-        assert "entitlement" == consumer["owner"]["contentAccessMode"]
+        with pytest.raises(
```

Comment:
since the tests use unittest and not pytest, I'd avoid adding also pytest here; please use `self.assertRaises()` instead